staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation.go

```diff
@@ -92,34 +93,55 @@ func validator(s *schema.Structural, isResourceRoot bool) *Validator {
 }
 
 // Validate validates all x-kubernetes-validations rules in Validator against obj and returns any errors.
-func (s *Validator) Validate(fldPath *field.Path, sts *schema.Structural, obj interface{}) field.ErrorList {
```

Comment:
I'm ignoring changes to this file and assuming this will be rebased on top of https://github.com/kubernetes/kubernetes/pull/108482

Reply:
Yes it is going to be rebased on #108482 . This PR is only for updating the budget. Thanks